packages/conf-rust-wasm/conf-rust-wasm.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Léo Andrès <contact@ndrs.fr>"
+authors: "Léo Andrès <contact@ndrs.fr>"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+build: [
+  ["sh" "-c" "rustc --print target-list | grep -q 'wasm32-unknown-unknown'" ]
+]
+depends: [
+  "conf-rust" {build}
+]
+depexts: [
+  ["libstd-rust-dev-wasm32"] { os-family = "alpine" }
```

Comment:
I don't think this pacakge exists here either. According to pkgs dot org, this package name is only used in debian/ubuntu: https://pkgs.org/search/?q=libstd-rust-dev-wasm32